File: TravelBois2/WebApp/Models/Misc/OcenaLeta.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.Models.Misc
{
    public class OcenaLeta
    {
		[Key]
		[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
		public string ID { get; set; }
		[Required]
		public string UserID { get; set; }
		[Required]
		public int Value { get; set; }
		[Required]
		public int IdLeta { get; set; }
	}
}

File: TravelBois2/WebApp/ClientApp/src/app/entities/misc/ocena-leta.ts
export class OcenaLeta {
  userID: string;
  value: number;
  idLeta: number;

  constructor(value: number, userID: string, idLeta: number) {
    this.value = value;
    this.userID = userID;
    this.idLeta = idLeta;
  }
}
